harmony-kotlin/src/commonMain/kotlin/com.harmony.kotlin/common/TypesUtilities.kt

```diff
@@ -0,0 +1,49 @@
+package com.harmony.kotlin.common
+
+import com.benasher44.uuid.Uuid
+import com.benasher44.uuid.uuid4
+import kotlin.random.Random
+
+const val ANY_ITEMS_COUNT = 5
+
+fun <T> getSome(itemsCount: Int = ANY_ITEMS_COUNT, builder: (index: Int) -> T): List<T> {
```

Comment:
Well, usually I put this kind of functions in a file called "MockAssets.kt", and all ObjectMothers are using this to generate random objects. The point is that, in Harmony this is not added exclusively for testing, so that's why we picked that name. Do you have a suggestion?

Reply:
Ok, no real suggestion. I thought something like RandomValues but I am not sure because the getSome does not generate anything random. Let's keep it as it is